tests/test_datasets/test_combination.py

```diff
@@ -7,29 +9,122 @@
 from tests import cases
 
 
+def _triple_tensor_to_set_of_triples(triples: torch.Tensor) -> Set[Tuple]:
+    return {tuple(tr.tolist()) for tr in triples}
```

Comment:
This should be equivalent to 
https://github.com/pykeen/pykeen/blob/c34d4016bc979acb17b88ba3a20d0caa93427bf5/src/pykeen/utils.py#L1262-L1264

Maybe you can also directly use
https://github.com/pykeen/pykeen/blob/c34d4016bc979acb17b88ba3a20d0caa93427bf5/src/pykeen/utils.py#L1267-L1269